feedback.html

```diff
@@ -0,0 +1,181 @@
+    </div>
+    <div class="navbar-dark text-white">
+        <div class="container">
+            <nav class="navbar px-0 navbar-expand-lg navbar-dark">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                    <div class="navbar-nav">
+                        <a href="index.html" class="pl-md-0 p-3 text-decoration-none text-light">
+                            <h3 class="bold"><span class="color_danger">LAKSHYA</span><span class="color_white">CTF</span></h3>
+                        </a>
+                    </div>
+                    <div class="navbar-nav ml-auto">
+                        <a href="index.html" class="p-3 text-decoration-none text-white bold">Home</a>
+                        <a href="about.html" class="p-3 text-decoration-none text-light bold">About</a>
+                        <a href="hackerboard.html" class="p-3 text-decoration-none text-light bold">Hackerboard</a>
+                        <a href="login.html" class="p-3 text-decoration-none text-light bold">Login</a>
+                        <a href="register.html" class="p-3 text-decoration-none text-light bold">Register</a>
+                    </div>
+                </div>
+            </nav>
+        </div>
+    </div>
+    <div class="jumbotron bg-transparent mb-0 pt-3 radius-0">
+        <div class="container">
+            <div class="row hackerFont">
+                <div class="col-xl-8">
+                    <h1 class="display-1 bold color_white content__title mb-4">FEEDBACK<span class="vim-caret">&nbsp;</span></h1>
+                    <p class="text-grey text-spacey hackerFont lead mb-4">
+                        Tell us about your experience!
+                    </p>
+                    <form class="needs-validation" novalidate>
+                        <div class="row mb-4">
+                            <div class="col-md-6">
+                                <label for="easeOfAccess">Ease of Access</label>
+                                <select class="custom-select d-block w-100" id="easeOfAccess">
+                                    <option value="">Select Rating</option>
+                                    <option>Excellent</option>
+                                    <option>Average</option>
+                                    <option>Poor</option>
+                                    <option>N/A</option>
+                                </select>
+                                <div class="invalid-feedback">Please choose a rating.</div>
+                            </div>
+                            <div class="col-md-6">
+                                <label for="interface">Interface</label>
```

Comment:
Change to: How was the User Interface?
Options: 
- Easy to understand
- Just Right
- Difficult to navigate through.